functions/src/modules/Users.functions.ts

```diff
@@ -0,0 +1,89 @@
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+import GoogleSheet from '../services/GoogleSheets';
+import { DataSnapshot } from 'firebase-functions/lib/providers/database';
+import { withDefault } from '../utils/parsers';
+
+const db = admin.database();
+
+enum RegistrationColumns {
+  Details = 'Details',
+  Status = 'Status',
+  Timestamp = 'Timestamp',
+  Name = 'Name',
+  Country = 'Country',
+  EmailAddress = 'Email Address',
+  PhoneNumber = 'Phone Number',
+  WhatsApp = 'WhatsApp',
+  Languages = 'Languages',
+  Services = 'Services',
+  Experience = 'Experience',
+  Influencer = 'Influencer',
+  RecommendedBy = 'Recommended By',
+}
+
+enum Roles {
+  CR = 'CR',
+  FC = 'FC',
+  QC = 'QC',
+  TE = 'TE',
+  SE = 'SE',
+  SQR = 'SQR',
+}
+
+enum Decision {
+  Yes = 'Yes',
+  No = 'No',
+}
+
+export const importUserRegistrationData = functions.https.onRequest(
+  async (req: functions.Request, res: functions.Response) => {
+    const gsheets: GoogleSheet = new GoogleSheet(
+      functions.config().user.spreadsheet_id,
+      'Registrations'
+    );
+
+    const registrationRows = await gsheets.getRows();
+
+    const readyForDatabaseUpdate = registrationRows.map((elem: any) => {
+      return {
+        notes: withDefault(elem[RegistrationColumns.Details]),
+        status: elem[RegistrationColumns.Status],
+        timestamp:
+          new Date(elem[RegistrationColumns.Timestamp]).getTime() / 1000,
+        name: elem[RegistrationColumns.Name],
+        location: elem[RegistrationColumns.Country],
+        emailAddress: elem[RegistrationColumns.EmailAddress],
+        phoneNumber: elem[RegistrationColumns.PhoneNumber],
+        whatsApp: elem[RegistrationColumns.WhatsApp],
+        languages: elem[RegistrationColumns.Languages].split(', '),
```

Comment:
Languages are also boolean-valued objects, not arrays.

Reply:
Ok, already changed.